### PR TITLE
Add configuration to silence console warnings

### PR DESCRIPTION
**Pull request by digitalsadhu**
Thanks for the great mixin!

Just scratching a little itch here, keen to be able to turn the console warnings off.

### CODE REVIEW

File: es6/time-stamp.js [hunk 1]
```diff
@@ -9,16 +9,17 @@ export default (Model, bootOptions = {}) => {
     updatedAt: 'updatedAt',
     required: true,
     validateUpsert: false, // default to turning validation off
+    silenceWarnings: false
   }, bootOptions);
 
   debug('options', options);
 
-  if (!options.validateUpsert && Model.settings.validateUpsert) {
+  if (!options.silenceWarnings && !options.validateUpsert && Model.settings.validateUpsert) {
```
Comment (clarkbw) [inline es6/time-stamp.js]: This is making the control statement a little too complex.  It also has the effect that the next line `Model.settings.validateUpsert = false;` won't be run if you turn on silenceWarnings.
Perhaps just create a `function warn(options, msg)` and replace the console.warn calls with that.

Comment (digitalsadhu) [inline es6/time-stamp.js]: 👍

**Comment by clarkbw** [comment]
🎆 💯 Awesome, thanks!
Once last request.  Can you add a note to the README about the option?

**Comment by digitalsadhu** [comment]
Ok, hopefully thats it. :)

**Comment by digitalsadhu** [comment]
Yay merge, thanks! 
Any chance you can publish a new version soon?

**Comment by clarkbw** [comment]
Yes, I've been traveling, but I'll push it out today. 🎉

**Comment by clarkbw** [comment]
pushed v3.4.0 11e6b4f4cb61bbf041c18a568998970c9df25240

**Pull request merged**